pollux/lib/core/src/main/scala/io/iohk/atala/pollux/core/repository/CredentialDefinitionRepositoryInMemory.scala

```diff
@@ -0,0 +1,97 @@
+package io.iohk.atala.pollux.core.repository
+
+import io.iohk.atala.pollux.core.model.*
+import io.iohk.atala.pollux.core.model.error.CredentialRepositoryError.*
+import io.iohk.atala.pollux.core.model.schema.CredentialDefinition
+import io.iohk.atala.shared.models.WalletAccessContext
+import zio.*
+
+import java.util.UUID
+
+class CredentialDefinitionRepositoryInMemory(
+    storeRef: Ref[Map[UUID, CredentialDefinition]]
```

Comment:
This in-memory is not tenant-aware. It could be fine, but may bring surprise later on. We can simply add wrapped `Ref` like in `PresentationRepositoryInMemory` with small code change.

https://github.com/input-output-hk/atala-prism-building-blocks/blob/1025f8faded420635441625e4f1ef9a31c9c0771/pollux/lib/core/src/main/scala/io/iohk/atala/pollux/core/repository/PresentationRepositoryInMemory.scala#L18.

Reply:
ok, let me take a look